crypto/evp/p5_crpt2.c

```diff
@@ -62,21 +62,6 @@ int ossl_pkcs5_pbkdf2_hmac_ex(const char *pass, int passlen,
 
```

Comment:
This "fail open" `rv` can also go. Either `return EVP_KDF_derive(...) == 1;` at the end or
```C
if (EVP_KDF_derive(kctx, out, keylen, params) != 1)
    return 0;
return 1;
```

Reply:
moved the rv assignment since we need to free the context regardless.

Reply:
Bah. Sorry.